parm/metplus/GridStat_ensprob_RETOP.conf

```diff
@@ -105,149 +103,130 @@ GRID_STAT_DESC = NA
 #
-# String to search for in the foreast input files for forecast variable
-# 1.
+# List of forecast and corresponding observation fields to process.
 #
-# Note:
-# This is the name of the field in the NetCDF file(s) created by MET's
-# gen_ens_prod tool.  This tool reads in the grib2 file(s) (in this case
-# of forecasts) and outputs NetCDF file(s) in which the array names 
-# consist of the value of fieldname_in_met_output plus a suffix that
-# specifies additional properties of the data in the array such as the
-# level, the type of statistic, etc.  In this case, this suffix is 
-# "_L0_ENS_FREQ_ge20".  Thus, below, FCST_VAR1_NAME must be set to the value
-# of fieldname_in_met_output with "_L0_ENS_FREQ_ge20" appended to it.
-# The same is true of other forecast fields below except the suffix is
-# different.
+# FREQ
+# Process as probability
 #
-#FCST_VAR1_NAME = RETOP_L0_ENS_FREQ_ge20
 FCST_VAR1_NAME = {{fieldname_in_met_output}}_L0_ENS_FREQ_ge20
 FCST_VAR1_LEVELS = L0
-#
-# List of thresholds to evaluate for each name/level combination for
-# forecast variable 1.
-#
 FCST_VAR1_THRESH = ==0.1
-
-#FCST_GRID_STAT_FILE_TYPE =
-#
-# String to search for in the observation input files for observation
-# variable 1.
-#
-# Note:
-# This is the name of the field in the grib2 observation file.  Thus,
-# it should not be set to {{fieldname_in_met_output}} because the
-# value of fieldname_in_met_output is in general not the same as the
-# name of the field in the grib2 observation file (although it can be
-# for certain fields).  If you do and it doesn't match, you may get an
-# error like this from METplus:
-#   ERROR  : VarInfoGrib2::set_dict() -> unrecognized GRIB2 field abbreviation ...
-# The same is true of other observation fields below.
-#
 OBS_VAR1_NAME = EchoTop18
-#
-# List of levels to evaluate for observation variable 1.  Must be the
-# same length as FCST_VAR1_LEVELS.
-#
 OBS_VAR1_LEVELS = Z500
-#
-# List of thresholds to evaluate for each name/level combination for
-# observation variable 1.
-#
-OBS_VAR1_THRESH = >=20.0
-OBS_VAR1_OPTIONS = censor_thresh = lt-20.0; censor_val = -20.0; cnt_thresh = [ >15 ]; cnt_logic = UNION; convert(x) = x * 3280.84 * 0.001;
+OBS_VAR1_THRESH = ge20
+OBS_VAR1_OPTIONS = censor_thresh = lt-20.0;
+                   censor_val = -20.0;
+                   cnt_thresh = [ >15 ];
+                   cnt_logic = UNION;
+                   convert(x) = x * 3280.84 * 0.001;
 
-#FCST_VAR2_NAME = RETOP_L0_ENS_FREQ_ge30
 FCST_VAR2_NAME = {{fieldname_in_met_output}}_L0_ENS_FREQ_ge30
 FCST_VAR2_LEVELS = L0
 FCST_VAR2_THRESH = ==0.1
-
 OBS_VAR2_NAME = EchoTop18
 OBS_VAR2_LEVELS = Z500
-OBS_VAR2_THRESH = >=30.0
-OBS_VAR2_OPTIONS = censor_thresh = lt-20.0; censor_val = -20.0; cnt_thresh = [ >15 ]; cnt_logic = UNION; convert(x) = x * 3280.84 * 0.001;
-
-#FCST_VAR3_NAME = RETOP_L0_ENS_FREQ_ge40 
-FCST_VAR3_NAME = {{fieldname_in_met_output}}_L0_ENS_FREQ_ge40 
+OBS_VAR2_THRESH = ge30
```

Comment:
Is there a reason that all of these different thresholds are considered separate variables here, vs with wind for example where different thresholds are specified for the same variable?

Reply:
@mkavulich I noticed this as well but decided not to explore it in too much depth for the purposes of this PR.  So I don't understand things completely yet, but here are a couple of hints:
1) The wind case(s) you noticed are in the **deterministic** PointStat files, i.e. `PointStat_(ADP)[SFC|UPA].conf`, whereas this file is for processing of **ensemble probabilistic** fields.  You can also see the thresholds grouped together for **deterministic** RETOP processing in `GridStat_REFC.conf`, where there are lines like [this](https://github.com/ufs-community/ufs-srweather-app/pull/973/files#diff-7caf53927728a2c4a5575d0f0f77e61c75471d1e985265e1aa06dee778159a4eR134):
  ```
  FCST_VAR1_THRESH = {{field_thresholds}}
  ```
  and `{{field_thresholds}}` is a list of values set in the ex-script, e.g. set to 'ge20, ge30, ge40, ge50' for RETOP.

2) Here, a specific forecast array named `{{fieldname_in_met_output}}_L0_ENS_FREQ_ge30` is being read in from the NetCDF file generated by GenEnsProd (where `{{fieldname_in_met_output}}` is equal to RETOP).  Note that the string `ge30` is hard-coded in this name.  If `OBS_VAR2_THRESH` were set to a list of thresholds instead, then we would somehow need to specify a list of array names in `FCST_VAR2_NAME` (as well as for `OBS_VAR2_NAME` and possibly other variables).  I don't know if METplus is capable of that.  Maybe it is, but I didn't want to explore further.

Maybe @michelleharrold has a better answer.  If there is a more compact way to express what we want done, I'd use it.